### DISCUSSION TITLE
quasar with StoryBook, Vue StyleGuidist, or ?

### DISCUSSION THREAD

**Discussion by flowt-au** [quasarframework/quasar#13147] [General - Components / Directives / etc]
Hi. I have a growing list of custom Quasar components and am thinking that something like StoryBook or Vue StyleGuidist or something similar might be useful for both development and documentation.

The only info I have found seems to be old and / or are questions about how to make them play nice with Quasar.

Is anyone successfully using these tools with Quasar 2? If so, which one(s)?

Thanks,
Murray

**Comment by fprijate** [comment]
Hi
Look here [Histoire Quasar](https://github.com/fprijate/histoire-quasar).
Not finished yet.
Open it on Stackblitz (button at the bottom,best working in Chrome)

**Comment by flowt-au** [comment]
@fprijate I will check it out. Thanks! :-)

**Comment by jclaveau** [comment]
I achieved a working Storybook 7 configuration with Quasar 2.12 and scss support : https://github.com/quasarframework/quasar/issues/11654#issuecomment-1618295838

It's hacky but it works (IMHO, big parts of it would better belong to Quasar's code base).

Histoire looks nice also, lighter but with less features (Mainly Chromatic)
